packages/incidentportal/src/router/index.ts

```diff
@@ -25,6 +25,7 @@ const routes: Array<RouteRecordRaw> = [
     meta: {
       title: IncidentPortalConfig.appTitle
     },
+    props: true,
```

Comment:
Should this not be dynamic?

Reply:
This config uses Vue's boolean mode. 

> When props is an object, this will be set as the component props as-is. Useful for when the props are static.

https://router.vuejs.org/guide/essentials/passing-props.html#boolean-mode